### DISCUSSION TITLE
Best practice to read LONG internet articles within koreader?  Self hosted wallabag?

### DISCUSSION THREAD

**Discussion by arooni** [koreader/koreader#11252] [General]
I remember setting this up years ago when framabag existed.  

Is wallabag still the best use case?  Sounds like I need to self host it on a VPS if I don't want to pay monthly for it? 

Thanks!

**Comment by arooni** [comment]
I wonder how framabag compares to "send to kindle" for long internet articles in terms of formatting?

**Reply by Frenzie** [reply]
Wallabag is still pretty good. I've always disliked that it strips comments but oh well. :-)

But yeah, either pay someone or put it on a VPS afaik.

**Reply by Frenzie** [reply]
Or just on your computer if you don't care if it's always available.
